pkg/util/test/util.go

```diff
@@ -288,6 +288,42 @@ func ValidateStorageCluster(
 
 	pxEndpoint = fmt.Sprintf("%s:%d", svc.Spec.ClusterIP, servicePort)
 
+	if err := validatePodsByName(cluster.Namespace, "portworx-pvc-controller", timeout, interval); err != nil {
+		return err
+	}
+
+	if cluster.Spec.Stork != nil && cluster.Spec.Stork.Enabled {
```

Comment:
In case of stork, lighthouse and autopilot we should validate the image in the pod/deployment is same as storage cluster.
For instance stork deployment should have `cluster.Spec.Stork.Image`

Reply:
@thiguetta you can refer to this PR https://github.com/portworx/spawn/pull/518 in CreatePortworxStorageClusterSpec() in cmd/provision_px_k8s.go file to see what I'm fetching from the StorageCluster yaml and implement same checks here as well.

Reply:
Done